bootstrap/helpers/socialite.php

```
@@ -44,6 +44,17 @@ function get_socialite_provider(string $provider)
         return Socialite::driver('zitadel')->setConfig($zitadel_config);
     }
 
+    if ($provider == 'oidc') {
+        $oidc_config = new \SocialiteProviders\Manager\Config(
```

PHPMD (2.15.0)

Missing class import via use statement (line '48', column '28'). (undefined)

(MissingImport)